### DISCUSSION TITLE
define a region for boxes

### DISCUSSION THREAD

**Discussion by hmsuchier** [OFFLINE-GmbH/oc-boxes-support#77] [Ideas and feature requests]
Hi, 
 Is it possible to add the notion of "region" for boxes? In order to display certain boxes in a given template zone, and other boxes elsewhere, etc?

**Comment by tobias-kuendig** [comment]
This is currently not possible. The Boxes Editor has just one main area you can edit. 

I'll add this to our backlog and see how complex the implementation would be.

https://github.com/OFFLINE-GmbH/oc-boxes-support/issues/78
